Customizing/global/plugins/Services/Repository/RepositoryObject/ReportCompanyGlobal/classes/class.ilObjReportCompanyGlobal.php

```diff
@@ -275,7 +281,7 @@ private function addDatePeriodFilter($query)
 					'		OR hucs.end_date = \'0000-00-00\''.PHP_EOL.
 					'		OR hucs.end_date = \'-empty-\')'.PHP_EOL.
 					' AND hucs.end_date <= '.$this->gIldb->quote($end, "date").PHP_EOL.
-					' AND ( hc.type <> \'Selbstlernkurs\''.PHP_EOL.
+					' AND ( hc.type NOT IN (\'Selbstlernkurs\')'.PHP_EOL.
```

Comment:
Hier war mal die Praxisbegleitung mit drin. Dann kam die Anweisung, dass sie hier gar nicht auftauchen soll. Mit dem Nachsatz "In Zukunft kann Sie aber da eventuell auftauchen". Deshalb hab ich das so gelassen.